### DISCUSSION TITLE
Example DAG for Snowflake Provider is broken

### DISCUSSION THREAD

**Discussion by jpmugizi** [apache/airflow#25190] [General]
### What do you see as an issue?

When navigating to the [example DAG](https://github.com/apache/airflow/tree/main/airflow/providers/snowflake/example_dags) hyperlink provided in the [Airflow API documentation](https://airflow.apache.org/docs/apache-airflow-providers-snowflake/1.3.0/connections/snowflake.html), we land at this error 
![image](https://user-images.githubusercontent.com/99832980/180033260-4e4b1fa1-91ce-40a1-a46b-a50662344bc9.png)

### Solving the problem

Replace the link in the doc : https://github.com/apache/airflow/tree/main/airflow/providers/snowflake/example_dags with the new working one.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
This is an old 1.3.0 version. I believe it is not worth to correct it. All the new versions have good links. However if you feel like it feel free to make a PR fixing it. You will find the htlm page here: https://github.com/apache/airflow-site/tree/main/docs-archive
